src/Microsoft.DotNet.Compiler.Common/ProjectContextExtensions.cs

```diff
@@ -34,25 +37,40 @@ public static void MakeCompilationOutputRunnable(this ProjectContext context, st
 
             if (context.TargetFramework.IsDesktop())
             {
+                // On full framework, copy all dependencies to the output path
                 exporter
                     .GetDependencies()
                     .SelectMany(e => e.RuntimeAssets())
                     .CopyTo(outputPath);
+                    
+                // Generate binding redirects
+                var outputName = context.GetOutputPathCalculator(outputPath).GetAssemblyPath(configuration);
```

Comment:
The tfm folder will have just compile time assets and the runtime folder will have everything https://gist.github.com/davidfowl/9931f6b9caa69b3b91dd#file-outputpaths-md